ios/RCCTabBarController.m

```diff
@@ -166,6 +166,10 @@ - (instancetype)initWithProps:(NSDictionary *)props children:(NSArray *)children
     id selectedIcon = tabItemLayout[@"props"][@"selectedIcon"];
     if (selectedIcon) {
       iconImageSelected = [RCTConvert UIImage:selectedIcon];
+      
+      if (!selectedButtonColor) {
```

Comment:
Can you please clarify this change?

Reply:
Sorry for long delay, without this fix it is not possible to show selected icon as is.

Without fix:
![without_fix](https://user-images.githubusercontent.com/858391/30242529-b4c0f73c-95a0-11e7-8ba8-ee912579269e.png)
With fix:
![with_fix](https://user-images.githubusercontent.com/858391/30242530-b4c37246-95a0-11e7-80be-1cea24ea9d66.png)

Reply:
The same for tabBarSelectedLabelColor.

Reply:
@guyca

Reply:
Please merge this. I have the same problem.